src/qmapshack/gis/rte/router/CRouterRoutino.cpp

```diff
@@ -248,14 +242,47 @@ void CRouterRoutino::buildDatabaseList()
                 continue;
             }
 
+            // qDebug() << "buildDatabase" << prefix;
+
 #ifdef Q_OS_WIN
             Routino_Database * data = Routino_LoadDatabase(dir.absolutePath().toLocal8Bit(), prefix.toLocal8Bit());
 #else
             Routino_Database * data = Routino_LoadDatabase(dir.absolutePath().toUtf8(), prefix.toUtf8());
 #endif
+            QVariantMap dmap;
```

Comment:
Why outside if clause? It's just used within the if clause context.

Reply:
Not applicable when re-arranging the if test as suggested later.